sdformat_to_mjcf/sdformat_to_mjcf/converters/geometry.py

```diff
@@ -98,6 +99,22 @@ def add_geometry(body, name, pose, sdf_geom):
     return geom
 
 
+def apply_surface_to_geometry(geom, sdf_surface):
+    """
+    Applies surface parameters from an SDFormat surface to an MJCF geom.
+
+    :param mjcf.Element geom: The MJCF geom to which the surface parameters
+            are applied.
+    :param sdformat.Surface sdf_surface: Surface object to be applied.
+    """
+
+    if sdf_surface is None:
```

Comment:
I don't know if we need to check if `sdf_surface` is None since a collision 
from a properly loaded SDFormat file would always have a default constructed `sdf::Surface` object. If you think it's necessary, do we also need to check if `sdf_surface.friction()` is `None`?

Reply:
I was following the example of `add_geometry`, which validates its input `sdf_geom` parameter, but I hadn't thought about whether that would actually be necessary. I think this would only be a problem if someone passed `None` explicitly into this method. I'll remove the logic.